### ISSUE TITLE
Disable vehicle UI from the server for client [New Feature]

### ISSUE THREAD

**Issue by AuD1tek** [SmartlyDressedGames/Unturned-3.x-Community#2339]
Could you add disabling the GUI of the vehicle (when the player gets into the vehicle) from the plugin.
Alternatively, add a new flag or flags to EPluginWingetFlags.

I really hope you add something like this! Thanks in advance :)

**Here is my implementation**
[PlayerLifeUI](https://pastebin.com/eCUmFYe8) *(pastebin)*
[EPluginWidgetFlags](https://pastebin.com/tUJSRdkU) *(pastebin)*

**Comment by z1net** [comment]
Good idea dude, i think about this 2 days ago, i want to write to Nelson with this problem, but im forgot this, thx u! Good luck

**Comment by QERT2002** [comment]
Yeah, very good idea, i think you can add this and a WeaponInfoBoxDisable flag in plugin

**Comment by DiFFoZ** [comment]
Duplicate of #1411

**Comment by z1net** [comment]
Ты тупой или что? Удали нахуй комментарий зайди в дс уебанище

**Comment by QERT2002** [comment]
> Duplicate of #1411

oh my god guy? are you stupid?

**Comment by SDGNelson** [comment]
Closing because the issues list was not really intended for feature requests. Sorry to close this discussion - I do WANT to fulfill all (most?) requests, but it is a matter of prioritization. Constantly being reminded of all the pending requests when checking issues is stressful. I will keep the request in mind. On GitHub maybe the Ideas section of Discussions would be the best place.
